Lib/test/test_dataclasses.py

```diff
@@ -3043,11 +3043,11 @@ def test_keyword_field_names(self):
     def test_non_identifier_field_names(self):
         for field in ['()', 'x,y', '*', '2@3', '', 'little johnny tables']:
             with self.subTest(field=field):
-                with self.assertRaisesRegex(TypeError, 'must be valid identifers'):
+                with self.assertRaisesRegex(TypeError, 'must be valid identifiers'):
                     make_dataclass('C', ['a', field])
-                with self.assertRaisesRegex(TypeError, 'must be valid identifers'):
+                with self.assertRaisesRegex(TypeError, 'must be valid identifiers'):
```

Comment:
```suggestion
                with self.assertRaisesRegex(TypeError, 'identifiers must be valid'):
```

Reply:
I think `must be valid identifiers` is better. That wording says that the text is not a a valid identifier. `identifiers must be valid` says that it's an identifier, but not a "valid" one, which doesn't mean anything.

Reply:
@ericvsmith so perhaps the appropriate suggestion would be `must be a valid identifier`? I'll update it accordingly. If the current phrasing is still preferable, let me know.

Reply:
@ericvsmith Thanks for the feedback. After looking more closely at the context, the current wording is more correct than my suggestion. `must be valid identifiers` looked incorrect when I was initially reviewing it. There might be a way to improve it, but `identifiers must be valid` is not.

Plus, as @terryjreedy mentioned in one of my other suggestions, it's probably best for this to primarily focus on uncontroversial typos rather than focus on the wording.